jonprl-mode.el

```diff
@@ -558,7 +558,21 @@ Invokes `jonprl-mode-hook'."
   ;; This is a customizable hook, hence the indirection.
   (add-hook 'after-save-hook 'jonprl-mode-run-after-save-hook t t)
   (yas-define-snippets 'jonprl-mode jonprl-vernacular-snippets)
-  (jonprl-update-operators))
+  (jonprl-update-operators)
+
+  (eval-after-load 'flycheck
+    '(progn
+       (flycheck-define-checker jonprl
+                                "A JonPRL proof checker."
```

Comment:
The indentation here seems a bit funky - doesn't `flycheck-define-checker` have an indentation spec in elisp? I don't think it blocks a merge, because this mode basically only exists for historical reasons and demos.